DXMainClient/Online/CnCNetManager.cs

```diff
@@ -260,6 +262,37 @@ private void DoChannelModesChanged(string userName, string channelName, string m
             channel.OnChannelModesChanged(userName, modeString);
         }
 
+        public void OnChannelListReceived(List<Tuple<string, string>> channelList)
+        {
+            Logger.Log("OnChannelListReceived called");
+
+            foreach (var nameTopic in channelList)
+            {
+                string channelName = nameTopic.Item1; 
```

Comment:
C# has builtin support to deconstruct a tuple, which gets rid of meaningless name like "Item1". 

(string name1, string name2) = tuple;


https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/deconstruct

Reply:
Good to know cheers. Have updated